.gitattributes

```diff
@@ -1,5 +1,5 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
-* text=auto
+* text eol=lf
```

Comment:
I think this is not a good idea. From my understanding, `auto` ensures that checked in files have LF line endings (therefore the repo has LF line endings). But on my local machine, I have the line ending that is appropriate for the system. Git will convert the line endings when commiting. 

Setting the `eol=lf`means that Windows developers will also have lf line endings on their local machine, which can have disadvantages. 

Therefore I suggest staying with `auto`.